plaid/client.py

```diff
@@ -37,7 +37,6 @@ class Client(object):
     def __init__(self,
                  client_id,
                  secret,
-                 public_key,
```

Comment:
this is a bit weird, right?

if you update python library, and you did not provide timeout before, the args woudl shift, and weirdt hings would happen right?

Reply:
I added a check for the environment here - https://github.com/plaid/plaid-python/pull/223/files#diff-80ef43562cf03f8dbeacd8ff6e7e4d4eR68

So now, if this library is updated to the next major version without making any code changes, users will get an error saying that the environment provided is not correct.